src/util/is_char_in_unicode_block.js

```diff
@@ -257,6 +261,7 @@ const unicodeBlockLookup: UnicodeBlockLookup = {
     // 'Tangut Components': (char) => char >= 0x18800 && char <= 0x18AFF,
     // 'Kana Supplement': (char) => char >= 0x1B000 && char <= 0x1B0FF,
     // 'Kana Extended-A': (char) => char >= 0x1B100 && char <= 0x1B12F,
+    // 'Small Kana Extension': (char) => char >= 0x1B130 && char <= 0x1B16F,
```

Comment:
Small Kana Extension and Symbols and Pictographs Extended-A consist of characters that should be given ideographic breaking and vertical layout, but for our lack of astral plane support: #4001.